src/provider/AbstractUnixProvider.php

```diff
@@ -9,6 +9,7 @@
 abstract class AbstractUnixProvider extends AbstractProvider
 {
     protected $cpuInfo;
+	protected $cpuInfoByLsCpu;
```

Comment:
Hi, why ``cpuInfoByLsCpu`` property is in AbstractUnixProvider while corresponding methods are in LinuxProvider? Can you move it there?